### ISSUE TITLE
Add Primary Source captions to Image alt-text

### ISSUE THREAD

**Issue by gjost** [denshoproject/encyc-tng#250]
This triggers a Wagtail accessibility validation exception and Articles cannot be saved in the UI without this. Save the editors a lot of copy-pasting.

**Comment by gjost** [comment]
Fixed as of branch `250-image-alt-text` commit 3f58e6f, merged into `develop` in commit 558a20d.

Thanks to Matthew at Torchbox for guidance and for wading through my migration code!
